### DISCUSSION TITLE
series/git-in-use/2023-02-04-git-tips---reset-a-branch-to-remote-status--git-in-use/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#43] [General]
# series/git-in-use/2023-02-04-git-tips---reset-a-branch-to-remote-status--git-in-use/

You got unexpected changes or commits on a branch. This is a tip for you to restore the branch to the fresh state of a remote branch

https://blog.vphong.com/series/git-in-use/2023-02-04-git-tips---reset-a-branch-to-remote-status--git-in-use/

**Comment by ba-ppp** [comment]
nice reset
